### PR TITLE
Hardcoding about page links for now

### PR DESCRIPTION
**Pull request by tyler-ball**
The real change is to add a `RELEASE_NOTES.md` document to Chef
Workstation that we manually update (like we do for ChefDK). Then
we include that into the Chef Workstation package, read it at runtime
and render it from markdown to HTML.

The package details can be parsed from the output of `chef -v` and
rendered to markdown as well.

The current release notes information is mostly copied from https://github.com/chef/chef-dk/pull/1797

Signed-off-by: tyler-ball <tball@chef.io>

🚧 Pending updates / completion of ChefDK release and release notes. Then we need to get a current channel build of Chef Workstation with that version of ChefDK so we can get current `chef -v` output

### CODE REVIEW

**Comment by tyler-ball** [comment]
These definitely need some styling love:

image

image

File: .expeditor/config.yml [hunk 1]
```diff
@@ -42,5 +42,5 @@ merge_actions:
         - "Expeditor: Skip All"
   # The markdown generator depends on the updated CHANGELOG.md so
   # should run after it
-  - bash:.expeditor/generate_markdown.sh:
-      only_if: built_in:update_changelog
+  # - bash:.expeditor/generate_markdown.sh:
+  #     only_if: built_in:update_changelog
```
Comment (jonsmorrow) [inline .expeditor/config.yml]: Shouldn't we still update the change log for app? And just not produce release notes from it?

File: assets/html/package_details.html [hunk 2]
```diff
@@ -5,6 +5,17 @@
...
+<body><h1 id="chef-workstation-package-details">Chef Workstation Package Details</h1>
+<p>The following is a list of included software and their versions:</p>
+<ul>
+<li>Chef Workstation: 0.2.27<ul>
+<li>chef-run: 0.2.0</li>
+<li>chef-client: 14.5.33</li>
+<li>delivery-cli: master (6862f27aba89109a9630f0b6c6798efec56b4efe)</li>
+<li>berks: 7.0.6</li>
+<li>test-kitchen: 1.23.2</li>
+<li>inspec: 2.2.112</li>
```
Comment (jonsmorrow) [inline assets/html/package_details.html]: wont this end up being 3.0 once dk gets released?

Comment (tyler-ball) [reply assets/html/package_details.html]: Yup! Good note, will need to update all these once ChefDK is built and I install a current channel build

File: .expeditor/config.yml (hunk 1 above)
Comment (tyler-ball) [reply .expeditor/config.yml]: `CHANGELOG.md` will still be getting updated by `built_in:update_changelog:` on line 38. By commenting out the lines I did we will not convert the local `CHANGELOG.md` into `assets/html/package_details.html`

**Pull request merged**